src/components/Dropdown/Dropdown.module.css

```diff
@@ -0,0 +1,139 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #DROPDOWN
+\*------------------------------------*/
+
+
+/**
+ * Dropdown used to select one option from a list of options.
+ */
+.dropdown {
+  position: relative;
+}
+
+/**
+ * Compact variant.
+ */
+.dropdown--compact {
+  width: min-content;
+}
+.dropdown-button--compact {
+  padding: var(--eds-size-half);
+}
+
+/**
+ * Label on top of the dropdown button to label the dropdown.
+ */
+.label {
+  @mixin eds-theme-typography-label-md;
+  color: var(--eds-theme-color-form-label-text);
+  margin-bottom: var(--eds-size-half);
+}
+
+/**
+ * Visually hides the label element if a aria-label is passed,
+ * but element is kept for screen reading accessibility.
+ */
+.sr-only {
+  @mixin visuallyHide;
+}
```

Comment:
Do we need this? Could we just put the `u-is-vishidden` class name on the element?

Reply:
I think this is needed for component functionality.
As for hiding via `u-is-vishidden` I think I would need to import that, which is the same as `@mixin visuallyHide` anyways iirc?

Reply:
How about we keep this but rename it to `u-is-vishidden` for consistency?

Reply:
Sounds good, will rename to `u-is-vishidden`